books/free-programming-books-subjects.md

```diff
@@ -273,7 +274,9 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Temporal Database Management](http://people.cs.aau.dk/~csj/Thesis/) - Christian S. Jensen
 * [The Theory of Relational Databases](http://web.cecs.pdx.edu/~maier/TheoryBook/TRD.html)
 
+### Data Analysis Cheatsheet
```

Comment:
I feel it is better to move this resource under python as it is python specific

Reply:
Okay, I'll move it under the python file in a separate PR, and close this one.